src/OrchardCore/OrchardCore.ReCaptcha.Core/TagHelpers/ReCaptchaTagHelper.cs

```diff
@@ -1,67 +1,128 @@
         public ReCaptchaMode Mode { get; set; }
 
-        public override void Process(TagHelperContext context, TagHelperOutput output)
+        /// <summary>
+        /// The two letter ISO code of the language the captcha should be displayed in
+        /// When left blank it will fall back to the default OrchardCore language
+        /// </summary>
+        [HtmlAttributeName("language")]
+        public string Language { get; set; }
+
+        public override async Task ProcessAsync(TagHelperContext context, TagHelperOutput output)
         {
-            var httpContext = _httpContextAccessor.HttpContext;
             var robotDetectors = _httpContextAccessor.HttpContext.RequestServices.GetServices<IDetectRobots>();
             var robotDetected = robotDetectors.Invoke(d => d.DetectRobot(), _logger).Any(d => d.IsRobot) && Mode == ReCaptchaMode.PreventRobots;
             var alwaysShow = Mode == ReCaptchaMode.AlwaysShow;
             var isConfigured = _settings != null;
 
             if (isConfigured && (robotDetected || alwaysShow))
             {
-                ShowCaptcha(output);
+                await ShowCaptcha(output);
             }
             else
             {
                 output.SuppressOutput();
             }
         }
 
-        private void ShowCaptcha(TagHelperOutput output)
+        private async Task ShowCaptcha(TagHelperOutput output)
         {
             output.TagName = "div";
             output.Attributes.SetAttribute("class", "g-recaptcha");
             output.Attributes.SetAttribute("data-sitekey", _settings.SiteKey);
             output.TagMode = TagMode.StartTagAndEndTag;
 
             var builder = new TagBuilder("script");
-            builder.Attributes.Add("src", _settings.ReCaptchaScriptUri);
+            var cultureInfo = await GetCultureAsync();
+
+            var settingsUrl = $"{_settings.ReCaptchaScriptUri}?hl={cultureInfo.TwoLetterISOLanguageName}";
+
+            builder.Attributes.Add("src", settingsUrl);
             _resourceManager.RegisterFootScript(builder);
         }
+
+        private async Task<CultureInfo> GetCultureAsync()
+        {
+            CultureInfo culture = null;
+
+            if (!string.IsNullOrWhiteSpace(Language))
+            {
+                try
+                {
+                    culture = CultureInfo.GetCultureInfo(Language);
+                }
+                catch (CultureNotFoundException)
+                {
+                    // if a developer makes a mistake, it is a warning
+                    _logger.LogWarning(T["Language with name; {0} not found", Language]);
+                }
+            }
+
+            if (culture == null)
+            {
+                // This should always return a valid culture
+                var ocCultureName = await _localizationService.GetDefaultCultureAsync();
+
+                try
+                {
+                    culture = CultureInfo.GetCultureInfo(ocCultureName);
```

Comment:
I don't think this will fail, because there's a fallback `CultureInfo.InstalledUICulture.Name`